EventListener/ReportSubscriber.php

```diff
@@ -229,9 +231,17 @@ private function alterSelect()
                     $fieldId = $this->extendedFields[$realField]['id'];
 
                     if (array_key_exists($realField, $this->fieldTables)) {
-                        $this->selectParts[$key] = $this->fieldTables[$realField]['alias'].'.value AS '.$fieldAlias;
+                        if ($aggregate) {
+                            $this->selectParts[$key] = preg_replace("/$string[1]/", $this->fieldTables[$realField]['alias'].'.value', $selectPart, 1);
```

Comment:
you might want to consider checking out [preg_quote](http://php.net/manual/en/function.preg-quote.php) to mak sure $string[1] wont cause issues?